core/src/main/resources/jenkins/telemetry/impl/SlaveToMasterFileCallableUsage/description.jelly

```diff
@@ -0,0 +1,13 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core">
+    Jenkins controllers construct a remote-procedure-call (RPC) channel to agents to instruct them to perform work.
+    This channel is bidirectional and in a handful of cases agents made requests of the controller.
+    This was always tricky to secure,
+    and <a href="https://www.jenkins.io/security/advisory/2021-11-04/" target="_blank">recently</a>
+    the category of usages which involved access to files was more tightly restricted than before;
+    Jenkins developers are considering disabling this kind of usage entirely.
+    Since it is difficult to determine via static analysis or even manual code inspection which plugins are using this system,
+    we are collecting information on how widely it is used.
+    The data includes names of Java classes mainly in Jenkins core and plugins as well as method names and line numbers.
+    It should not include the names of files being accessed or anything else not determined by versions of software components in use.
```

Comment:
Vague statement is vague.

Reply:
Vague in what way? What would you suggest?